packages/features/insights/context/FiltersProvider.tsx

```diff
@@ -60,7 +60,7 @@ export function FiltersProvider({ children }: { children: React.ReactNode }) {
     dateRange: [
       startTimeParsed ? dayjs(startTimeParsed) : dayjs().subtract(1, "week"),
       endTimeParsed ? dayjs(endTimeParsed) : dayjs(),
-      "w",
+      !startTimeParsed && !endTimeParsed ? "w" : null,
```

Comment:
Fixes the `dateRange` not carrying value of startTime and endTime from URL params leading to UX issues